Resources/Prototypes/Loadouts/loadout_groups.yml

```diff
@@ -601,7 +603,7 @@
   loadouts:
   - MimeSuspendersRed
   - MimeSuspendersBlack
-  
```

Comment:
Should probably untouch it.

Reply:
I really have no idea what git is yapping about because I swear its exactly as it is now... I just copy pasted old file and just added those two lines and it still says that exists

Reply:
Visual studio code, if that's what you're using, automatically removes trailing whitespaces, which isn't something that should be touched in this PR. You should be able to disable it in the user settings.

Reply:
Think I got it